packages/buendia-backup/data/usr/bin/buendia-backup

```diff
@@ -122,13 +122,10 @@ sleep 1  # ensure that keep has the uniquely lowest timestamp in the directory
 
 echo
 echo "Saving Buendia configuration..."
-tar cfz "$new_dir/buendia.tar.gz" \
-  --exclude '*.omod' \
-  /usr/share/buendia/counts \
-  /usr/share/buendia/distilled \
-  /usr/share/buendia/openmrs \
-  /usr/share/buendia/profiles \
-  /usr/share/buendia/site
+# Back up only those configuration directories which already exist; don't worry
+# about any which might be missing.
+config_paths=$(ls -d /usr/share/buendia/{counts,distilled,openmrs,profiles,site} || true)
+tar cfz "$new_dir/buendia.tar.gz" --exclude '*.omod' $config_paths
```

Comment:
I think the `--ignore-failed-read` option will solve this in a simpler way.  If it's easy & quick for you to test, let's go with that, otherwise just merge this as is.  Thanks!

Reply:
It was my understanding that `--ignore-failed-read` still returns a non-zero exit value; otherwise I'd have used it

Reply:
Also, I didn't `mkdir -p /usr/share/buendia/distilled` because technically that directory belongs to `buendia-dashboard` and I thought that package should manage it.

Reply:
Okay!